s4e-backend/pom.xml

```diff
@@ -20,13 +20,13 @@
     <properties>
         <jjwt.version>0.10.7</jjwt.version>
         <recaptcha-starter.version>2.3.1</recaptcha-starter.version>
-        <commonmark.version>0.13.1</commonmark.version>
-        <hibernate-types.version>2.9.0</hibernate-types.version>
+        <commonmark.version>0.14.0</commonmark.version>
+        <hibernate-types.version>2.9.3</hibernate-types.version>
         <poiooxml.version>4.1.1</poiooxml.version>
         <greenmail.version>1.5.11</greenmail.version>
         <slugify.version>2.4</slugify.version>
-        <springdoc-openapi.version>1.2.25</springdoc-openapi.version>
-        <awssdk.version>2.10.44</awssdk.version>
+        <springdoc-openapi.version>1.2.30</springdoc-openapi.version>
+        <awssdk.version>2.10.57</awssdk.version>
```

Comment:
how this happen? 
https://mvnrepository.com/artifact/software.amazon.awssdk/s3/2.10.56